app/blueprints/fund_builder/routes.py

```diff
@@ -72,8 +76,15 @@ def section(round_id):
         "round_id": str(round_id),
     }
     existing_section = None
+    # TODO there must be a better way than a pile of ifs...
     if request.args.get("action") == "remove":
```

Comment:
I have no strong feelings about this, but if you're interested, the pattern I tend to use is 
```
action_handlers = {
"remove" : lambda : delete_section_from_round(round_id=round_id, section_id=request.args.get("section_id"), cascade=True),
"move_up" : lambda : move_section_up(round_id=round_id, section_index_to_move_up=int(request.args.get("index"))),
...
}

try:
    action_handlers[request.args.get("action")]()
    return redirect(url_for("build_fund_bp.build_application", round_id=round_id))
except KeyError:
    # Some default handling or pass
```
The one benefit of the above is that the multiple ifs aren't executed (although the same could be achieved with if/elif etc). I'm not sure one is tidier than the other though!